app/http.php

```diff
@@ -708,11 +708,11 @@ function (string $runtimeId, string $payload, array $variables, int $timeout, st
                     throw new Exception('An internal curl error has occurred within the executor! Error Msg: ' . $error, 500);
                 }
 
-                if ($i === 4) {
+                if ($i === 49) {
                     throw new Exception('An internal curl error has occurred within the executor! Error Msg: ' . $error, 500);
                 }
 
-                \sleep(1);
+                \sleep(0.1);
```

Comment:
```suggestion
                \usleep(100000);
```

Same here, use `usleep` instead.